book/src/pong-tutorial/pong-tutorial-01.md

```diff
@@ -208,159 +197,54 @@ Then we call `.run()` on `game` which starts the game loop. The game will
 continue to run until our `SimpleState` returns `Trans::Quit`, or when all states
 have been popped off the state machine's stack.
 
-Try compiling the code now. You should be able to see the window already.
-The content of that window right now is undefined and up to the operating system.
-It's time to start drawing on it.
+Try compiling the code now. You should be able to see that application starts but nothing
```

Comment:
`You should be able to see the application start, but nothing will happen and your terminal will hang until you kill the process. This means that the core game loop is running in circles, and is awaiting tasks. Let's give it something to do by adding a renderer!`